OnlinerTests/PageObjects/CatalogPage.cs

```diff
@@ -0,0 +1,74 @@
+﻿using OpenQA.Selenium;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using WebElement = OnlinerTests.PageObjects.Basic.WebElement;
+
+namespace OnlinerTests.PageObjects
+{
+    public class CatalogPage : BasePage
+    {
+        public CatalogPage() : base()
+        {
+
+        }
+
+        private WebElement _cellphoneHeader => new WebElement(By.XPath("//h1[contains(text(),'Мобильные телефоны')]"));
```

Comment:
this is a property. so its name should be in camal case. fix here and everywhere else

Reply:
what should I fix here? cellphone is a single word, I don't think there is need to make p capital. And I think it is a field as it is hidden. (Don't want to be rude tho)

Reply:
private WebElement CellphoneHeader => new(...)